src/udiskslinuxblock.c

```diff
@@ -966,6 +966,21 @@ udisks_linux_find_child_configuration (UDisksDaemon *daemon,
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+static gchar *
+get_slave_sysfs_path (const gchar *sysfs_path)
+{
+  gchar *ret = NULL;
+  gchar **slaves;
+  slaves = udisks_daemon_util_resolve_links (sysfs_path, "slaves");
+  if (g_strv_length (slaves) == 1)
```

Comment:
I think this should also check `slaves != NULL`.

Reply:
Done